src/styles/shared/components/buttons/_index.scss

```diff
@@ -1,63 +1,16 @@
-@use "sass:map";
-
-@use "../../variables" as var;
-
-@use './_config' as *; // Bring variables into scope
 @use './mixins';
 
-// A button with only an icon and no label/text
-.IconButton {
-  @include mixins.Button(
-    $options: (
-      'colormap': $IconButton-colors,
-    )
-  ) {
-    @media (pointer: coarse) {
-      min-width: $touch-target-size;
-      min-height: $touch-target-size;
-    }
-  }
-  &--primary {
-    @include mixins.Button--variant(
-      $options: (
-        'colormap': $IconButton-colors--primary,
-      )
-    );
-  }
-}
-
 // A button with text, and optionally an icon
 .LabeledButton {
-  @include mixins.Button(
-    $options: (
-      'colormap': $LabeledButton-colors,
-      'withLayout': true,
-    )
-  );
-  &--primary {
-    @include mixins.Button--variant(
-      $options: (
-        'colormap': $LabeledButton-colors--primary,
-      )
-    );
-  }
+  @include mixins.LabeledButton;
```

Comment:
The responsibility of this (the "entry point" of `buttons`) is reduced to just output styles; mixins moved to `mixins`